src/libraries/System.IO.FileSystem/tests/FileStream/ctor_options.cs

```diff
@@ -162,21 +162,25 @@ public void PreallocationSize(FileAccess access, FileMode mode)
         [InlineData(FileMode.CreateNew)]
         public void WhenDiskIsFullTheErrorMessageContainsAllDetails(FileMode mode)
         {
-            const long tooMuch = 1024L * 1024L * 1024L * 1024L; // 1 TB
-
+            const long tooMuch = 1024L * 1024L * 1024L * 1024L * 1024L * 1024L; // 1 Exbibyte .. we are assuming this is not available
             string filePath = GetTestFilePath();
 
-            IOException ex = Assert.Throws<IOException>(() => CreateFileStream(filePath, mode, FileAccess.Write, FileShare.None, bufferSize: 1, FileOptions.None, tooMuch));
-            Assert.Contains(filePath, ex.Message);
-            Assert.Contains(tooMuch.ToString(), ex.Message);
-
-            // ensure it was NOT created
-            bool exists = File.Exists(filePath);
-            if (exists)
+            try
+            {
+                // not using Assert.Throws because in the event of failure, we want to dispose, so the next test isn't affected
+                Assert.Throws(() =>
```

Comment:
```suggestion
                Assert.Throws<IOException>(() =>
```
This would fix the build. You should also extract the other asserts from the try-catch.

I would consider reverting to the previous commit instead 😄.

Reply:
reverted to what it was originally as I think it's fine. can you signoff @Jozkee ?